quickstart/101-front-door-standard-premium/readme.md

```diff
@@ -2,298 +2,29 @@
-
-| Terraform Resource Type | Description |
-| - | - |
-| `azurerm_resource_group` | The resource group for all the deployed resources.|
-| `azurerm_cdn_frontdoor_profile` | The Front Door profile. |
-| `azurerm_cdn_frontdoor_endpoint` | The Front Door endpoint. |
-| `azurerm_cdn_frontdoor_origin_group` | The Front Door origin group. |
-| `azurerm_cdn_frontdoor_origin` | The Front Door origin, which refers to the App Service app. |
-| `azurerm_cdn_frontdoor_route` | The Front Door route. |
-| `azurerm_service_plan` | The App Service plan. |
-| `azurerm_windows_web_app` | The App Service app. |
-| `random_id` | Two random identifier generators to generate a unique Front Door endpoint resource name and App Service app name. |
+## Terraform resource types
+
+- [random_pet](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet)
+- [azurerm_resource_group](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group)
+- [random_id](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/id)
+- [azurerm_cdn_frontdoor_profile](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_profile)
+- [azurerm_cdn_frontdoor_endpoint](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_endpoint)
+- [azurerm_cdn_frontdoor_origin_group](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_origin_group)
+- [azurerm_cdn_frontdoor_origin](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_origin)
+- [azurerm_cdn_frontdoor_route](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_route)
+- [azurerm_service_plan](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/service_plan)
+- [azurerm_windows_web_app](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/windows_web_app)
 
 ## Variables
 
-| Name | Description | Default Value |
+| Name | Description | Default value |
 |-|-|-|
-| `location` | The location for all the deployed resources. | `westus2` |
-| `resource_group_name` | The name of the resource group. | `FrontDoor` |
-| `app_service_plan_sku_name` | The SKU for the App Service plan. | `S1` |
-| `app_service_plan_sku_tier_name` | The name of the tier that the App Service plan SKU is in. | `Standard` |
+| `resource_group_name_prefix` | Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription. | rg |
+| `resource_group_location` | Location of the resource group. | eastus |
+| `app_service_plan_sku_name` | The SKU for the plan. Possible values include: B1, B2, B3, D1, F1, I1, I2, I3, I1v2, I2v2, I3v2, I4v2, I5v2, I6v2, P1v2, P2v2, P3v2, P0v3, P1v3, P2v3, P3v3, P1mv3, P2mv3, P3mv3, P4mv3, P5mv3, S1, S2, S3, SHARED, EP1, EP2, EP3, WS1, WS2, WS3, Y1. | S1 |
```

Comment:
Out of curiosity, did you speak internally about whether it's a good idea to document the full list? App Service adds new SKUs semi-regularly so this list could go stale.

Reply:
@johndowns We did have that dialogue and it was decided that the benefits of having the information in the code (variables.tf and readme.md) outweigh the maintenance costs.